exampleCourse/questions/template/multiple-choice/fixed-correct/question.html

```diff
@@ -0,0 +1,10 @@
+<pl-question-panel>
+  <p>What does the acronym "CPU" stand for in Computer Science?</p>
```

Comment:
Nit, capital "Computer Science" felt weird to me. Open to other ways of phrasing this too.

```suggestion
  <p>What does the acronym "CPU" stand for in computing?</p>
```